kasa/module.py

```diff
@@ -49,38 +49,38 @@ class Module(ABC):
     IotTime: Final[ModuleName[iot.Time]] = ModuleName("time")
 
     # SMART only Modules
-    Alarm: Final[ModuleName[smart.AlarmModule]] = ModuleName("AlarmModule")
-    AutoOff: Final[ModuleName[smart.AutoOffModule]] = ModuleName("AutoOffModule")
+    Alarm: Final[ModuleName[smart.Alarm]] = ModuleName("Alarm")
+    AutoOff: Final[ModuleName[smart.AutoOff]] = ModuleName("AutoOff")
     BatterySensor: Final[ModuleName[smart.BatterySensor]] = ModuleName("BatterySensor")
     Brightness: Final[ModuleName[smart.Brightness]] = ModuleName("Brightness")
-    ChildDevice: Final[ModuleName[smart.ChildDeviceModule]] = ModuleName(
-        "ChildDeviceModule"
-    )
-    Cloud: Final[ModuleName[smart.CloudModule]] = ModuleName("CloudModule")
-    Color: Final[ModuleName[smart.ColorModule]] = ModuleName("ColorModule")
-    ColorTemp: Final[ModuleName[smart.ColorTemperatureModule]] = ModuleName(
-        "ColorTemperatureModule"
+    ChildDevice: Final[ModuleName[smart.ChildDevice]] = ModuleName("ChildDevice")
+    Cloud: Final[ModuleName[smart.Cloud]] = ModuleName("Cloud")
+    Color: Final[ModuleName[smart.Color]] = ModuleName("Color")
+    ColorTemperature: Final[ModuleName[smart.ColorTemperature]] = ModuleName(
+        "ColorTemperature"
     )
     ContactSensor: Final[ModuleName[smart.ContactSensor]] = ModuleName("ContactSensor")
-    Device: Final[ModuleName[smart.DeviceModule]] = ModuleName("DeviceModule")
-    Energy: Final[ModuleName[smart.EnergyModule]] = ModuleName("EnergyModule")
-    Fan: Final[ModuleName[smart.FanModule]] = ModuleName("FanModule")
+    Device: Final[ModuleName[smart.Device]] = ModuleName("Device")
```

Comment:
We should probably find a way to avoid overloading the `Device` naming one more time

Reply:
So maybe we should as an exception to the rule call the Device module `DeviceModule` instead of `Device`. What do you think?

Reply:
Ok renamed the `Device` module to `DeviceModule`